pkg/chunk/cassandra/storage_client.go

```diff
@@ -180,6 +182,15 @@ func (cfg *Config) setClusterConfig(cluster *gocql.ClusterConfig) error {
 			}
 		}
 	}
+
+	if cfg.HostSelectionPolicy == "RoundRobin" {
```

Comment:
**suggestion(non-blocking):** consider defining `RoundRobin` and `TokenAware` as constant enums. For example:

```go
const (
  SelectionPolicyRoundRobin = "RoundRobin"
  SelectionPolicyTokenAware = "TokenAware"
)
```